### DISCUSSION TITLE
외주 요청 정리

### DISCUSSION THREAD

**Discussion by KimDa99** [seong0929/Platypus#53] [Ideas]
# 캐릭터
- 사이즈 최소 (32 * 32 px) ~ 최대 (64 * 64 px), 권장 사이즈 (3550, 3550 px)
- 왼쪽을 바라보는 것을 기준
- 필요한 모션
  - Idle
  - 이동
  - 일반 공격
  - 스킬
  - 궁극기
  - 데스
  - 리스폰
  - 기타 임팩트

**Comment by KimDa99** [comment]
# 배경
- 로딩
- 홈
- 스테디움
- 전투씬 : 사바나(모래의 기운), 바다(물의 기운), 숲(풀의 기운)
- 우승씬??
+ 예상치 못한 것 3개

- 금액 기댓값
7.5만 * 10개  = 75만원 / 49일 (2달)

**Comment by seong0929** [comment]
- [외주 후보 이슈](https://github.com/seong0929/Platypus/issues/48)

**Comment by KimDa99** [comment]
# 배경 요청
### 공통
- 픽셀 사이즈: 512 * 288
- 전투 배경
- 배경 전체는 캐릭터가 돌아다닐 수 있음
- 쿼터뷰 시점
- 낮은 체도

## Sepia 세피아 (사막, 사바나, 모래) 

## Aqua 아쿠아 (바다, 물) 

## emerald 에메랄드 (숲, 풀, 정글)

**Reply by KimDa99** [reply]
안녕하세요. 게임 배경 일러스트 견적을 묻고 싶습니다.

1. 우선 배경 일러스트를 작업하시나요?

2. 배경 일러스트를 작업하신다면, 512*288 크기의 일러스트는 견적이 얼마일까요? 애니매이션은 없으며 + 상업적으로 사용할 것이고 + 수정 가능한 원본파일이 필요합니다.

3. 예상 작업일이 어떻게 될까요?

4. 원활한 작업을 위해서 어떠한 자료를 준비하면 좋을까요?

---
저희가 원하는 배경의 컨셉은 다음과 같습니다.

- 픽셀 사이즈: 512 * 288
- 전투 배경
- 배경 전체는 캐릭터가 돌아다닐 수 있는 쿼터뷰 시점
- 낮은 채도 (캐릭터들이 잘 보일 수 있도록)
- 모바일

- 3가지 배경 컨셉
  - Sepia 세피아 (사막, 사바나, 모래)
  - Aqua 아쿠아 (바다, 물)
  - emerald 에메랄드 (숲, 풀, 정글)
